src/components/templates/SearchTemplate/index.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import KaKaoMap from "@/components/molecules/KaKaoMap";
+import { MdSearch } from "react-icons/md";
+import React, { useRef, useState } from "react";
+import useToast from "@/hooks/useToast";
+
+function SearchTemplate() {
+  const ref = useRef<HTMLInputElement>(null);
+  const [place, setPlace] = useState("");
+  const [update, setUpdate] = useState(false);
```

Comment:
boolean 변수는 is를 접두사로 붙이면 좋습니다!
```suggestion
  const [isQpdate, setIsUpdate] = useState(false);
```